README.md

```diff
@@ -2,27 +2,31 @@ Apache TubeMQ
 ==============================================
 [![Build Status](https://travis-ci.org/apache/incubator-tubemq.svg?branch=master)](https://travis-ci.org/apache/incubator-tubemq)
 
-Apache TubeMQ (incubating) is a trillion-records-scale distributed messaging queue (MQ) system, focuses on data transmission and storage under massive data. Compared to many open source MQ projects, TubeMQ has unique advantages in terms of stability, performance, and low cost.
+[Apache TubeMQ](https://tubemq.apache.org)(incubating) is a trillion-records-scale distributed messaging queue (MQ) system, focuses on data transmission and storage under massive data. Compared to many open source MQ projects, TubeMQ has unique advantages in terms of stability, performance, and low cost.
 
+It offers a variety of features:
 
-Contact
--------
-
+* Pluggable transport protocols, such as TCP, SSL
+* Support big-data and streaming ecosystem integration
+* Message retroactivity by time or offset
+* Efficient pull and push consumption model
+* Flexible distributed scale-out deployment architecture
+* Feature-rich administrative dashboard for configuration
+* Authentication and authorization
 
-- Mailing lists
-
-| Name                                                                          | Scope                           |                                                                 |                                                                     |                                                                              |
-|:------------------------------------------------------------------------------|:--------------------------------|:----------------------------------------------------------------|:--------------------------------------------------------------------|:-----------------------------------------------------------------------------|
-| [dev@tubemq.apache.org](mailto:dev@tubemq.apache.org)     | Development-related discussions | [Subscribe](mailto:dev-subscribe@tubemq.apache.org)   | [Unsubscribe](mailto:dev-unsubscribe@tubemq.apache.org)   | [Archives](http://mail-archives.apache.org/mod_mbox/tubemq-dev/)   |
 
+Contact us
+-------
 
-- Issue management
-  [See JIRA](https://issues.apache.org/jira/browse/TubeMQ)
+- Mail list: **dev@tubemq.apache.org**. Mail to `dev-subscribe@tubemq.apache.org`, follow the reply to subscribe the mail list.
```

Comment:
After this change, it will be more troublesome for users to subscribe to the mailing list. I think it's fine to show the mailing list using the list now. Users only need to click the button to subscribe the mail.

Reply:
Can you provide a mature example for reference?

Reply:
I have changed back to the original mail list form